_pages/pages/documentation/build-scans.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Build Scans
+permalink: /pages/documentation/build-scans/
+layout: docs
+navigation: pages
+sidenav: pages-documentation
+---
+
+
+Builds Scans are a new experimental feature being offered by Pages.
+
+For each site build, users can see scan results which check their website for security vulnerabilities, accessibility violations, and other relevant information to help improve their site. These scans begin running after a build completes. When the scan finishes, the results are available as downloadable HTML files for easy sharing among team members. There are currently two scans available: ZAP Security Scan and Accessibility Scan. If you are interested in being part of the beta test of this new feature, please email us at pages-support@cloud.gov.
```

Comment:
We aren't going to scan each site build. Maybe something like `For production site builds...` or `Select site builds...`